Sources/KlaviyoCore/Networking/KlaviyoAPI.swift

```diff
@@ -40,7 +40,7 @@ public struct KlaviyoAPI {
             return .failure(.missingOrInvalidResponse(response))
         }
 
-        if httpResponse.statusCode == 429, httpResponse.statusCode == 503 {
+        if httpResponse.statusCode == 429 || httpResponse.statusCode == 503 {
```

Comment:
Hmm we didn't a have that failed?

Reply:
Are you asking why we did have a failing test?

Reply:
If yes, you are right, some unit test should have caught this. Cut a ticket to add some unit tests.

Reply:
Yah I'm surprised I thought I added one but maybe it didn't test things enough.

Reply:
nah I should have tested when I moved 429 and added 503. Have limited time this week so cut a ticket but hopefully we can add it soon.